### PR TITLE
kitchen garden

### PR DESCRIPTION
**Pull request by m-coding**
Please review this, @casey-collab

### CODE REVIEW

File: english/garden/beans.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Ah! Good choice.
```
Comment (casey-collab) [inline english/garden/beans.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/garden/carrots.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+What's up doc? Carrots contain beta-carotene.
```
Comment (casey-collab) [inline english/garden/carrots.md]: This file has also been added to a directory that already contained a page.

File: english/garden/choose.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+The garden is beautiful.
```
Comment (casey-collab) [inline english/garden/choose.md]: This file has also been added to a directory that already contained a page.

File: english/garden/tomatoes.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+What a treat!
```
Comment (casey-collab) [inline english/garden/tomatoes.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by m-coding** [comment]
Please review my changes, @casey-collab

File: english/garden/beans.md (hunk 1 above)
Comment (casey-collab) [inline english/garden/beans.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/garden/carrots.md (hunk 2 above)
Comment (casey-collab) [inline english/garden/carrots.md]: This file has also been added to a directory that already contained a page.

File: english/garden/choose.md (hunk 3 above)
Comment (casey-collab) [inline english/garden/choose.md]: This file has also been added to a directory that already contained a page.

File: english/garden/tomatoes.md (hunk 4 above)
Comment (casey-collab) [inline english/garden/tomatoes.md]: This file has also been added to a directory that already contained a page.

Comment (casey-collab) [inline english/garden/tomatoes.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/garden/beans.md (hunk 1 above)
Comment (casey-collab) [inline english/garden/beans.md]: This page is also unreachable.

File: english/garden/carrots.md (hunk 2 above)
Comment (casey-collab) [inline english/garden/carrots.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by m-coding** [comment]
See more changes, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :smile:

**Pull request merged**